repositories.md

```diff
@@ -50,6 +52,31 @@ The following properties are supported:
 
 It is not necessary to specify the format of the index -- this will be auto-detected so long as the format is one of those supported by the plugin. The index file may optionally be compressed with gzip. 
 
+OSGiRepositories
```

Comment:
The `OSGiRepository` is a fixed index repository. It is just the replacement for `FixedIndexRepo`. So this heading seems wrong.